frontend/src/components/explorer/MapViewer.vue

```diff
@@ -141,6 +141,12 @@ export default {
       this.handleQueryParamsWatch(newQuery, oldQuery);
     },
   },
+  async beforeCreate() {
+    const copyright = document.getElementById('footer-copyright');
+    copyright.className = 'column';
+    const logos = document.getElementById('footer-logos');
+    logos.style.display = 'none';
+  },
```

Comment:
If two layouts are needed, it is preferred to do this purely in CSS, even if it would imply two alternative designs.